### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by heather-everton**

### CODE REVIEW

File: models/Issue_State.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```
Comment (jesseparent) [inline models/Issue_State.js]: We should not allow null

File: models/Issue_User.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Issue_user.init(
+    {
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+    //   name: {
```
Comment (jesseparent) [inline models/Issue_User.js]: Remove commented out lines

File: models/Project.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```
Comment (jesseparent) [inline models/Project.js]: Don't allow null

File: models/Project_State.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```
Comment (jesseparent) [inline models/Project_State.js]: Don't allow null

File: models/Role.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```
Comment (jesseparent) [inline models/Role.js]: Don't allow null

File: models/Team.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```
Comment (jesseparent) [inline models/Team.js]: Don't allow null

File: models/User_State.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```
Comment (jesseparent) [inline models/User_State.js]: Don't allow null

**Review by jesseparent — COMMENTED** [review]
Looks good overall - too many fields that should be required but are allowing nulls

File: models/User_State.js (hunk 7 above)
Comment (heather-everton) [reply models/User_State.js]: name: {
        type: DataTypes.STRING,
        allowNull: false
      }

File: models/Team.js (hunk 6 above)
Comment (heather-everton) [reply models/Team.js]: name: {
        type: DataTypes.STRING,
        allowNull: false
      }

File: models/Role.js (hunk 5 above)
Comment (heather-everton) [reply models/Role.js]: name: {
        type: DataTypes.STRING,
        allowNull: false
      }

File: models/Project_State.js (hunk 4 above)
Comment (heather-everton) [reply models/Project_State.js]: name: {
        type: DataTypes.STRING,
        allowNull: false
      }

File: models/Project.js (hunk 3 above)
Comment (heather-everton) [reply models/Project.js]: name: {
        type: DataTypes.STRING,
        allowNull: false
      }

File: models/Issue_User.js (hunk 2 above)
Comment (heather-everton) [reply models/Issue_User.js]: Removed

File: models/Issue_State.js (hunk 1 above)
Comment (heather-everton) [reply models/Issue_State.js]: name: {
        type: DataTypes.STRING,
        allowNull: false
      }

**Review by jesseparent — APPROVED** [review]
Looks great!

**Pull request merged**